ansible/inventory/group_vars/apps.yaml

```diff
@@ -74,25 +74,25 @@ messaging:
     contact_email: test@email.com
 
 compute:
-  key_pair_name: sandpit-key
+  key_pair_name: densikat-sandpit
   inbound_from_bastion_host_security_group: sg-a5ac6cc2
 
 s3:
   data_bucket_name: aem-stack-builder
 
 library:
-  aem_aws_stack_provisioner_version: 1.1.1
-  aem_orchestrator_version: 1.0.0
+  aem_aws_stack_provisioner_version: 1.1.2
+  aem_orchestrator_version: 1.0.1
```

Comment:
Can't default to 1.0.1 yet since it's yet to be released. This is pending for now until the fix for https://github.com/shinesolutions/aem-orchestrator/issues/30 is released as 1.0.1 .